ang/civicase/CaseListTable.js

```diff
@@ -29,13 +30,98 @@
     (function init () {
       bindRouteParamsToScope();
       initiateWatchers();
-
-      $scope.casePlaceholders = $scope.filters.id ? [0] : _.range($scope.page.size);
+      initSubscribers();
       getCases();
+      getAllCasesforSelectAll();
+      $scope.casePlaceholders = $scope.filters.id ? [0] : _.range($scope.page.size);
     }());
 
+    /**
+     * Initiate subscribers
+     */
+    function initSubscribers () {
```

Comment:
Private functions should be defined after public functions. JS refactoring wiki needs to be followed.